site/docs/v1/tech/account-management/authenticator-two-factor.adoc

```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
-title: Add TOTP Factor
-description:  How to add TOTP Authentication on the User and Account Levels
+title: Add Authenticator Factor
+description:  How to add Authenticator Authentication on the User and Account Levels
```

Comment:
Here's where you would add the `redirect_from` key value pair that I mentioned above.